### PR TITLE
Match Firebase Function for US Launch

### PR DESCRIPTION
**Pull request by iritush**
…s with the changes needed for the front end integration (aka US launch). The match function includes the day of week support

### CODE REVIEW

File: functions/src/match/scheduledMatch.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      today = "wednesday";
+      break;
+    case 4:
+      today = "thursday";
+      break;
+    case 5:
+      today = "friday";
+      break;
+    case 6:
+      today = "saturdat";
```
Comment (AlanPedro) [inline functions/src/match/scheduledMatch.js:139]: saturday here

File: functions/src/config/constants.js [hunk 2]
```diff
@@ -27,10 +27,26 @@ const TIME_PERIODS = {
...
+  "12": "12:00-14:00",
+  "14": "14:00-16:00",
+  "16": "16:00-18:00",
+  "18": "18:00-20:00",
+  "20": "20:00-22:00",
+  "22": "22:00-00:00",
+  "0": "00:00-02:00",
+  "2": "02:00-04:00",
+  "4": "04:00-06:00",
+  "6": "06:00-08:00"
```
Comment (AlanPedro) [inline functions/src/config/constants.js:42]: the time periods will need to match what is in the front end
let me get them now

Comment (AlanPedro) [reply functions/src/config/constants.js:42]: Spot on you had that already 👏

File: functions/src/match/scheduledMatch.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+      break;
+    case 7:
```
Comment (AlanPedro) [inline functions/src/match/scheduledMatch.js:141]: I think that Sunday is the first day and is Index = 0

File: functions/src/match/scheduledMatch.js (hunk 1 above)
Comment (iritush) [reply functions/src/match/scheduledMatch.js:139]: Thanks!!

**Comment by iritush** [comment]
Basic functionality is in.

**Pull request merged**